static/documentation/8.1.0/b10/bower/font-awesome/css/font-awesome.css

```
@@ -0,0 +1,3 @@
+version https://git-lfs.github.com/spec/v1
```

Stylelint (17.13.0)

[error] Unknown word https (CssSyntaxError)

(CssSyntaxError)